src/components/message/Message.tsx

```diff
@@ -46,11 +52,27 @@ const Message: FC<MessageProps> = ({ message, prevMessage, sx = {} }) => {
     e.stopPropagation()
   }
 
+  const messageRef = useRef<HTMLDivElement>(null)
+  const isTextFiltered = filteredMessages.some(
+    (filteredMessage) => filteredMessage.toLowerCase() === text.toLowerCase()
+  )
+  useEffect(() => {
+    if (filteredIndex >= 0 && filteredIndex < filteredMessages.length) {
```

Comment:
maybe check that filteredIndex > 0 instead of filteredIndex >= 0?

Reply:
In this case the index can be 0, because we are searching in an array